6. Developer Guides/REST API/Users/resetAvatar.md

```diff
@@ -0,0 +1,34 @@
+---
+order: 40
+---
+
+# User Reset Avatar
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/users.createToken` | `yes` | `POST` |
```

Comment:
Good catch, mind changing this? I apologise for forgetting about this pull request